.aws/task-definition-template.json

```diff
@@ -11,6 +11,10 @@
         {
           "name": "SENTRY_DSN",
           "valueFrom": "/tis/trainee/user-management/sentry/dsn"
+        },
+        {
+          "name": "PROFILE_CREATED_QUEUE_URL",
+          "valueFrom": "/tis/trainee/${environment}/queue-url/profile-created"
```

Comment:
We're not creating a profile so we don't want to send a profile create notification to this queue.
Instead we should send to `tis-trainee-sync-queue-prod`, which is badly named but that's the queue that `tis-sync` processes for data requests.

Reply:
Thank you. `/tis/trainee/sync/${environment}/queue-url` is used instead, that refer to `tis-trainee-sync-queue-prod`